clouddq/classes/dq_configs_cache.py

```diff
@@ -149,6 +150,25 @@ def get_row_filter_id(self, row_filter_id: str) -> dq_row_filter.DqRowFilter:
         )
         return row_filter
 
+    def get_reference_columns_id(
+        self, reference_columns_id: str
+    ) -> dq_reference_columns.DqReferenceColumns:
+        reference_columns_id = reference_columns_id.upper()
+        try:
+            reference_columns_record = self._cache_db["reference_columns"].get(
+                reference_columns_id
+            )
+        except NotFoundError:
+            error_message = (
+                f"Reference Column ID: {reference_columns_id} not found in 'reference_columns' config cache:\n"
+                f"{pformat(list(self._cache_db.query('select id from reference_columns')))}"
```

Comment:
add a line in the error message to explain what is being returned by this query.

Reply:
Updated the error message.